lighthouse.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+docker compose pull
+docker-compose -f docker-compose.yml -f docker-compose.ci.yml up -d
+
+echo "Waiting for backend to start at https://localhost.simplereport.gov/api/health"
+http_response=0
+polls=0
+while [[ $http_response != "200" && $polls -lt 360 ]]; do
+  ((polls++))
+  sleep 1
+  http_response=$(curl -skL -w "%{http_code}" "https://localhost.simplereport.gov/api/health")
+done
+if [[ $http_response -ne 200 ]]; then
+  echo 'Backend never started. Exiting...'
+  exit 1
+fi
+echo 'Backend started!'
+echo
+
+FACILITY_ID=$(docker exec db psql -qtA -v ON_ERROR_STOP=1 -p 5432 -U simple_report_migrations simple_report -c "select internal_id from simple_report.facility limit 1;")
+echo "Our facility id is: $FACILITY_ID."
+
+docker-compose -f docker-compose.yml -f docker-compose.ci.yml up
```

Comment:
Running in line 4 with `-d` doesn't expose the logs to `lhci`, so we can go ahead and start things up and get our facility id that we need to build urls for testing. Running `docker-compose ... up` here in line 24 attaches to the containers that were started up above in line 4, and the server logs are exposed to `lhci` which can then detect the server start and run the tests.

Reply:
Question: Does this show all previous logs that were run or only new logs from this point moving forward?

Reply:
I believe it streams all the logs, @alismx can you confirm?

Reply:
It'll stream all the logs.